src/utils/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Here we will be dispatching API actions
+ */
+import { message } from "antd";
+import axios from "axios";
+
+// We need to set default config for axios
+axios.defaults.baseURL = "https://fake-news-detector-backend.herokuapp.com";
+axios.defaults.headers.post['Access-Control-Allow-Origin'] = '*'
+axios.defaults.headers.post['accept'] = "application/json"
+axios.defaults.headers.post["Content-Type"] =
+  "application/json";
```

Comment:
```suggestion
axios.defaults.headers.post = { 
  'Access-Control-Allow-Origin': '*',
  'accept': 'application/json',
  'Content-Type': 'application/json',
}
```
We can move this into a single object rather having separate definition